src/mappings_explorer/site_builder.py

```diff
@@ -1364,6 +1364,25 @@ def build_attack_pages(projects: list, url_prefix: str, breadcrumbs: list):
                 breadcrumbs=breadcrumbs,
                 non_mappables=non_mappables,
             )
+            # Build technique pages that don't have mappings to fix any linking errors
+            for technique in non_mappables:
```

Comment:
I was a little thrown off by the use of the term "non-mappable", which I thought applied to security capabilities and not ATT&CK techniques. Looking at the code above this I can see that this is doing the right thing, but for future improvements let's come up with a different term, e.g. "unmapped" versus "non-mappable".

Reply:
@mehaase that's a good point! I changed the name of those variables for clarity